account_banking_nl_rabo_mt940/account_banking_nl_rabo_mt940.py

```diff
@@ -1,8 +1,9 @@
 # -*- coding: utf-8 -*-
+"""Implement parser for MT940 files - Rabobank dialect."""
 ##############################################################################
 #
 #    OpenERP, Open Source Management Solution
-#    This module copyright (C) 2014 Therp BV (<http://therp.nl>).
+#    Copyright (C) 2015 Therp BV <http://therp.nl>.
```

Comment:
Next time don't remove the year but can add the current year, e.g.

    Copyright (C) 2014-2015

Just leave it for now